timereport/curl_localhost.sh

```diff
@@ -19,3 +19,7 @@ else
   'body': 'token=fake_token&team_id=CHANGEME&team_domain=codelabsab&channel_id=CA8THJML6&channel_name=development&user_id=U2FGC795G&user_name=test.user&command=%2Fno-wsgi&text=$2+$3+$4:$5&response_url=https%3A%2F%2Fhooks.slack.com%2Fcommands%2FT2FG58LDV%2F491076166711%2FbVUlrKZrnElSOBUqn01FoxNf&trigger_id=490225208629.83549292471.860541eab9e9c3c6d7464ea2e979c7a5'
   }" $1
 fi
+
+# test "today"
+curl -X POST -d "{'body':
+'token=OwjK95k89vMbtbyvhvLZkXNl&team_id=CHANGEME&team_domain=codelabsab&channel_id=CA8THJML6&channel_name=development&user_id=U2FGC795G&user_name=kamger&command=%2Fno-wsgi&text=add+vab+today&response_url=https%3A%2F%2Fhooks.slack.com%2Fcommands%2FT2FG58LDV%2F491076166711%2FbVUlrKZrnElSOBUqn01FoxNf&trigger_id=490225208629.83549292471.860541eab9e9c3c6d7464ea2e979c7a5'}"
```

Comment:
Is this not sensitive information?

Reply:
good point! not sure!

Reply:
actually is the same token we have in test_data.py

Reply:
should probably change that everywhere then even if it's not in use